output.proto

```diff
@@ -71,6 +71,13 @@ message SystemHealth {
   map<string, Node> nodes = 2;
 }
 
+message OnlineRecalibrationEvent {
+  google.protobuf.Timestamp timestamp = 1;
+  string topic = 2;
+  Vector3 translation = 3;
+  Vector3 axis_rotation = 4;
```

Comment:
it should have the rotation angle, right?

Reply:
Here I used the euler angles, so 3 values should be enough for the rotation? Do you mean that we should use quaternions instead?

Reply:
ah, sorry, I though it was another definition which specify the rotation axis and rotation angle. It's fine then to use Euler angle.